src/Authenticator/Result.php

```diff
@@ -50,7 +50,7 @@ class Result implements ResultInterface
     /**
      * Sets the result status, identity, and failure messages
      *
-     * @param null|array|\ArrayAccess $data The identity data
+     * @param null|array|\ArrayAccess|mixed $data The identity data
```

Comment:
These `mixed` types needed to be added since we check for this and manually throw an exception

Reply:
Seems like we forgot to add types to this class when updating for 5.x. I would just add proper types to the props/methods and bump the plugin version as required to minor/major.

Reply:
We did not miss it, this is 2.x and therefore CakePHP 4
Or would you not update stan of the 2.x version of this plugin?

Reply:
I wouldnt. Only cake5 ones IMO.